cluster/common.sh

```diff
@@ -347,7 +347,7 @@ function find-release-tars() {
 
   # This tarball is used by GCI, Ubuntu Trusty, and CoreOS.
   KUBE_MANIFESTS_TAR=
-  if [[ "${KUBE_OS_DISTRIBUTION:-}" == "trusty" || "${KUBE_OS_DISTRIBUTION:-}" == "gci" || "${KUBE_OS_DISTRIBUTION:-}" == "coreos" ]]; then
+  if [[ "${OS_DISTRIBUTION:-}" == "trusty" || "${OS_DISTRIBUTION:-}" == "gci" || "${OS_DISTRIBUTION:-}" == "coreos" ]]; then
```

Comment:
Now that `OS_DISTRIBUTION` defaults to `gci`, one doesn't have to set `KUBE_OS_DISTRIBUTION=gci` any more to start a GCI cluster, and the old logic will start to fail in this case.